### DISCUSSION TITLE
Welcome to geocore-community Discussions!

### DISCUSSION THREAD

**Discussion by jonyo** [geodesicsolutions-community/geocore-community#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vicos59** [comment]
Howdy @jonyo,

Be happy to help out anyway I can do testing, minor coding, docs.

A good place to start might be to get https://geodesicsolutions.org/ SEO'd so people looking for the software or support have a better chance of finding the site via a search engine. I can tackle this. I also noticed that the version-to-version upgrade docs don't work (https://geodesicsolutions.org/update-instructions/). Is this a code issue or are we just missing the pages? If code, I might be able to take a look.

Some folks have been finding bugs (seems related to new PHP releases) and even posting fixes in the other forums. I suggested they post them here as a new issue, but, github may be a new beast to many; I'm not fluent myself. Might need some coaxing and a link or 2 to some Github 101 pages.

Finally, I am still running GeoCore MAX 7.5.1 and I need to first get it moved to a new server. Would you be able to gen up the unlock script you mentioned earlier? After that, will I be able to upgrade to the current version using the current code? or will have to install a fresh copy and then massage the data from the old database to the new database?

**Reply by jonyo** [reply]
Hey that was quick!  I just discovered this feature earlier today on Github 😆 

> A good place to start might be to get https://geodesicsolutions.org/ SEO'd so people looking for the software or support have a better chance of finding the site via a search engine. I can tackle this.

Sure I'd welcome any work you can contribute!  I literally did the minimal to get it not looking completely ugly though that is subjective 😆 ...  The website source is open as well if you haven't seen it over at https://github.com/geodesicsolutions-community/website - let me know if you need any help getting started with how to fork and do PR's.

Like this project, I have the website source set up to use docker to test changes locally really easily.  If you do a PR, I'll check it out and see it in action.

> I also noticed that the version-to-version upgrade docs don't work (https://geodesicsolutions.org/update-instructions/). Is this a code issue or are we just missing the pages? If code, I might be able to take a look.

It's working for me?  For instance, after typing in 7.0.0 and selecting GeoCore it loads this up:

https://geodesicsolutions.org/update-instructions/?product=GeoCore&version=7.0.0&method=wizard&tplUpdate=export

What specifically wasn't working on the update instruction generator?

>  Some folks have been finding bugs (seems related to new PHP releases) and even posting fixes in the other forums. I suggested they post them here as a new issue, but, github may be a new beast to many; I'm not fluent myself. Might need some coaxing and a link or 2 to some Github 101 pages.

I did see one about the admin cron page not working, and I already fixed that in the codeset (even used what was posted as a solution).

> Finally, I am still running GeoCore MAX 7.5.1 and I need to first get it moved to a new server. Would you be able to gen up the unlock script you mentioned earlier? After that, will I be able to upgrade to the current version using the current code? or will have to install a fresh copy and then massage the data from the old database to the new database?

Hmm, making an unlock for that old of a version would be a lot of time...  Instead this is what I'd suggest:

* Copy your files over to the new server.
* At this point it won't work.  But your old server is still going I think right?  So not too much of a rush.
* Do the update, using either the 18.02.0 file set if you have it with the license file patch on top to unlock it.  Or you can try the community edition, I won't be able to do it for you but I can walk you through it.  That might be the better option but it is more complicated because I still need to get "pre-built" releases set up.  Right now you have to run `composer install` to have it load all the libraries you need.  Anyways, so when you do the update, it should work since the files you are updating to won't have the license stuff on it.

**Reply by jonyo** [reply]
By the way, I'm actually looking into how to make pre-built releases now.  So with any luck, maybe some time this weekend there will be a zip you can download that will already have all the files built (`composer install` already run).  No promises though 😄

**Reply by vicos59** [reply]
>>Instruction Generator

My bad, it is working when you type in the version number. I remembered it having a dropdown of available versions and never thought to type in a version #.

-- 7.5.1 Upgrade

I had a test domain running on the old server and I moved it to the new server and it runs (Fortunately I still have PHP 5.4 on both servers). I was just looking for an unlocker as insurance. No worries since it is a lot of work. I will just try upgrading the test copy on the new server and do a fresh install and take it from there.

-- Site update for SEO.

I'll make some changes to add verbiage which addresses any of the keywords people might use when searching for the old Geodesic solutions site. We should probably get the site set up with Google and Bing Webmaster. I am well versed with this.

**Reply by vicos59** [reply]
Do you think you could get the previous owner to reassign the Facebook page to one or more people affiliated with the new open source version? or at least make a new post that points people to the new site?

https://www.facebook.com/GeodesicSolutions/

Business pages can have one or more admins/editors assigned to them. The admin/editors need to be a normal FB user account. At least that is how my business page works.

**Reply by jonyo** [reply]
I don't think I could get ahold of any  of the current admins on that existing FB page, it took forever to get ahold of one of the owners for the source code permission alone and I don't think they even realize the FB page exists 😆 

It might be better to create a new page, call it GeoCore CE or something.  Feel free to make one if you want and we can link to it on the website.

By the way, I managed to make a release!  It is already built so nothing needed other than what it says to do here (I updated these instructions for the community edition):

https://geodesicsolutions.org/wiki/installation/zipped/start

Here is the release, the download you want will be under "assets", the file `geocore-ce.zip`: https://github.com/geodesicsolutions-community/geocore-community/releases/tag/v1.0.0-beta.1

**Reply by vicos59** [reply]
I'll see if there is a way to claim it. If not, I'll set up a new business page and make us both admins. I already messaged a few people who had posted on the FB page looking for help just to let them know this resource is here.

I'll try out the new build tomorrow. I just setup a test domain for it on a CentOS server.

**Comment by jonyo** [comment]
Wow, maybe we can use this "discussions" space as an official space for the software...  It looks like it could be a really good replacement for a forum.

**Reply by vicos59** [reply]
It is nice to keep everything together.

**Comment by vicos59** [comment]
The new Facebook page for GeoCore CE is at:

https://www.facebook.com/GeoCoreCE

I'll work on some simple graphics when I get time, or pay someone on fiverr a few bucks to do a logo.

@jonyo, if you can follow the page on FB, I'll assign you as an admin so we can bus-proof the page.

**Reply by vicos59** [reply]
@jonyo, I can no longer find a way to see the list of followers to the FB page, so I added you manually. I hope I got the right guy...to be sure I added you as an editor. Once we confirm I got the right guy, I'll upgrade you to full admin. You're the one with an '.R.' in the middle?

**Reply by jonyo** [reply]
Yup, that's me, I accepted the invite.  So as long as you only invited one person, you got the right one.  Thanks!

**Reply by vicos59** [reply]
You R admin now.
